style-dictionary/transforms/size/px-to-rem.js

```diff
@@ -4,14 +4,18 @@ module.exports = (StyleDictionary) => {
   StyleDictionary.registerTransform({
     name: 'size/px-to-rem',
     type: 'value',
+    transitive: true,
     matcher(prop) {
       return prop.attributes.category === 'size';
     },
     transformer(prop) {
       // update what this is divided by if body font-size changes in rei-cedar
-      const num = (parseFloat(prop.value, 10) / 10);
-      const unit = num !== 0 ? 'rem' : ''
-      return `${num}${unit}`;
-    },
+     if (prop.value.indexOf('rem') === -1){
```

Comment:
This check is necessary to ensure that the conversion to rems doesn't happen more than once. This happens with transitive values.